scripts/test-runner.sh

```diff
@@ -47,8 +47,13 @@ if [[ $OS_TYPE != "fedora" ]]; then
     $PKG_MAN -y install epel-release
 fi
 
+# TEMPORARY WORKAROUND
+# TODO: Remove when a fixed official beakerlib version is released
+yum -y install https://sumsal.cz/dev/beakerlib-1.11centos-99.fc23.noarch.rpm
```

Comment:
Even though we use only a one little function, which consist of 4 lines, there are several other patches, which need to be applied, so this function could work properly. I hate this solution as much as you, but without that, we have several other not-so-nice solutions:

1) Store the patches in this repo and apply them before testing
2) Store the built RPM in the repo instead of somewhere on an external infrastructure

As much as I'd like to avoid any of these options, we'll have to pick one of them, otherwise the incredibly slow release cycle of beakerlib would block our progress.